mindmap/static/js/src/mindmapEdit.js

```diff
@@ -0,0 +1,19 @@
+/* Javascript for MindMapXBlock. */
+function MindMapXBlock(runtime, element) {
+
+  $(element).find('.save-button').click(function () {
+    var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+    var data = {
+      display_name: $(element).find('input[name=mindmap_display_name]').val(),
+    };
+    $.post(handlerUrl, JSON.stringify(data)).done(function (response) {
```

Comment:
Follow this advise to show an alert to the user
https://github.com/eduNEXT/xblock-mindmap/pull/2/files#r1276842559

Reply:
Please delete notifications alert and add a comment with a TOD

`// TODO add notifications`